tests/scadnano_tests.py

```diff
@@ -3544,6 +3544,97 @@ def test_color_specified_with_integer(self) -> None:
         actual_color_hex = d.strands[0].color.to_json_serializable(False)
         self.assertEqual(expected_color_hex, actual_color_hex)
 
+    def test_non_parallel_helices_when_using_single_helix_group(self) -> None:

```

Comment:
I didn't check the unit tests carefully, but make sure they are covering all the cases we outlined. (both Helix and Helix group have pitch specified, or neither, or one but not the other).